### DISCUSSION TITLE
Outermost pixels cannot be used

### DISCUSSION THREAD

**Discussion by B3N64** [cc-tweaked/CC-Tweaked#1717] [Q&A] [answered]
Outermost pixels cannot be used but can change color depending on which background color was used when writing next to them. They also do not change when the background color is changed. Is there a workaround?

Code:
term.setCursorPos(1,1)
term.blit("\x80\x81\x82\x83\x84\x85\x86\x87\x88\x89\x8a\x8b\x8c\x8d\x8e\x8f\x90\x91\x92\x93\x94\x95\x96\x97\x98\x99\x9a\x9b\x9c\x9d\x9e\x9f", "ffffffffffffffffffffffffffffffff", "0123456789abcdef0000000000000000")
term.blit("\x80\x81\x82\x83\x84\x85\x86\x87\x88\x89\x8a\x8b\x8c\x8d\x8e\x8f\x90\x91\x92\x93\x94\x95\x96\x97\x98\x99\x9a\x9b\x9c\x9d\x9e\x9f", "00000000000000000000000000000000", "ffffffffffffffffffffffffffffffff")

term.setCursorPos(1,3)
term.blit("Test1", "fffff", "abcde")

term.setCursorPos(1,4)
term.write("Test2")

term.setCursorPos(1,5)
term.setBackgroundColor(2048)
term.write("Test3")

term.setCursorPos(1,6)
term.setBackgroundColor(128)
term.write("Test4")

![2024-02-09_18 20 31](https://github.com/cc-tweaked/CC-Tweaked/assets/91955311/62e4a75c-75b5-4671-8a02-e3a2cf9ad67c)
![2024-02-09_20 05 30](https://github.com/cc-tweaked/CC-Tweaked/assets/91955311/d72fa19d-4b24-4a27-8207-81b1689f8ab2)

**Comment by Wojbie** [comment]
Duplicate: #722 
Related: https://github.com/dan200/ComputerCraft/issues/51

Oh wait this is a question not an issue.. what is the question again?

**Reply by B3N64** [reply]
Is there a solution?

**Reply by Wojbie** [reply, marked as answer]
I am afraid not. It will always set those border pixels to be same colour as background color of the closest character. This logic was added way before drawing characters were implemented and yes it is a known issue that does not have a solution yet. Most users just avoid using drawing characters next to border. You can look into threads i linked to see previous discussions related to this.
